src/open_inwoner/accounts/tests/test_contact_views.py

```diff
@@ -45,11 +41,16 @@ def test_contact_list_only_show_personal_contacts(self):
         self.assertEqual(response.status_code, 200)
         self.assertNotContains(response, self.contact.first_name)
 
+    def test_list_shows_pending_invitations(self):
+        existing_user = UserFactory()
+        self.user.contacts_for_approval.add(existing_user)
+        response = self.app.get(self.list_url, user=self.user)
+        self.assertContains(response, existing_user.first_name)
+
     def test_contact_filter(self):
-        begeleider = ContactFactory(
-            contact_user__contact_type=ContactTypeChoices.begeleider,
-            created_by=self.user,
-        )
+        begeleider = UserFactory(contact_type=ContactTypeChoices.begeleider)
+        self.user.user_contacts.add(begeleider)
```

Comment:
please check also that unwanted `contacts_for_approval` and `pending_invitations` are not shown